server/src/application/application.entity.ts

```diff
@@ -12,6 +19,9 @@ export class Application extends CustomBaseEntity {
   @PrimaryGeneratedColumn('uuid', { name: 'application_id' })
   id: string;
 
+  @VersionColumn()
+  version: number;
```

Comment:
Instead of adding it everywhere, could you add this to the BaseEntity so that it gets added along with Date columns?